src/biokbase/narrative/jobs/job.py

```diff
@@ -132,37 +130,21 @@ def from_attributes(cls, **kwargs):
         children (list): applies to batch parent jobs, Job instances of this Job's child jobs
         """
         # reconstruct the ee2 job state object
-        ee2_state = kwargs.get("ee2_state", {})
-        job_input = ee2_state.get("job_input", {})
-        narr_cell_info = job_input.get("narrative_cell_info", {})
-
-        narr_cell_info.update(
-            {
-                key: kwargs[key]
-                for key in NARR_CELL_INFO_ATTRS
-                if key in kwargs and key not in narr_cell_info
-            }
-        )
-        job_input.update(
-            {
-                **{
-                    _attr_to_ee2(key): kwargs[key]
-                    for key in JOB_INPUT_ATTRS
-                    if key in kwargs and key not in job_input
-                },
-                "narrative_cell_info": narr_cell_info,
-            }
-        )
-        ee2_state.update(
-            {
-                **{
-                    key: kwargs[key]
-                    for key in STATE_ATTRS
-                    if key in kwargs and key not in ee2_state
-                },
-                "job_input": job_input,
-            }
-        )
+        narr_cell_info = {
```

Comment:
Removed feature where you can instantiate `Job` using attributes _and_ state

Reply:
Good. That might lead to confusion.